lib/Parse/Lexer.cpp

```diff
@@ -283,6 +283,8 @@ void Lexer::formToken(tok Kind, const char *TokStart, bool MultilineString) {
       return Piece.isComment();
     });
     for (auto End = LeadingTrivia.end(); Iter != End; Iter++) {
+      if (Iter->getKind() == TriviaKind::Backtick)
+        break;
```

Comment:
Can we whitelist comment kinds instead of blacklisting backsticks? That way makes more sense to me.

Reply:
We have to whitelist all kind of comments and whitespaces.

Reply:
ok, can we add some comments here why backtick is different? It's kind of missing connection that here `CommentLength` means more than comments in syntax tree.

Reply:
Added!

Reply:
Thanks!